src/main/java/com/google/firebase/auth/internal/FirebaseTokenVerifier.java

```diff
@@ -82,11 +82,7 @@ private FirebaseTokenVerifier(Builder builder) {
     checkArgument(!Strings.isNullOrEmpty(builder.method), "method must be set");
     this.projectId = builder.projectId;
     this.shortName = builder.shortName;
-    if ("aeiouAEIOU".indexOf(shortName.charAt(0)) < 0) {
-      this.articledShortName = "a " + shortName;
-    } else {
-      this.articledShortName = "an " + shortName;
-    }
+    this.articledShortName = addArticle(shortName);
```

Comment:
Maybe make this `prefixWithIndefiniteArticle`?

Reply:
Done